src/components/layout/__tests__/Header.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import { Header } from '../Header';
+
+
+const setUp = () => {
+  const wrapper = mount(
+    <Router>
+      <Header/>
+    </Router>,
+  );
+
+  return {
+    wrapper,
+  };
+};
+
+describe('Register User test::', () => {
+  const { wrapper } = setUp();
+  it(' render header component', () => {
+    const headerMenu = wrapper.find('.header-menu');
+
+    expect(headerMenu.exists())
+      .toBe(true);
+  });
+
+  // it(' handle menu click on item', () => {
+  //   const pathName = { name: 'home' };
+  //   wrapper.instance().handleItemClick({}, pathName);
```

Comment:
Please remove all instances of commented out code

Reply:
Done